### ISSUE TITLE
Wrong content type in APIStar 0.3 when rendering html page

### ISSUE THREAD

**Issue by pavelholvi** [encode/apistar#285]
<img width="710" alt="screen shot 2017-09-15 at 1 53 34 am" src="https://user-images.githubusercontent.com/22292899/30459337-b82a63ce-99b8-11e7-8b49-c12cb4898430.png">

I've tested on 0.2 and 0.3, seems like we treat this as `application/json`, a bit tricky for me to debug but I believe the problem is somewhere here:
https://github.com/encode/apistar/blob/526fe9aeecfeb03737e86cf7999b6129982846c2/apistar/hooks.py#L68

**Comment by pavelholvi** [comment]
Ok so problem is actually here:
https://github.com/encode/apistar/blob/master/apistar/renderers.py#L55

https://github.com/encode/apistar/blob/master/apistar/renderers.py#L56

Was there any reason to leave out the HTML renderer?

**Comment by pavelholvi** [comment]
```
from apistar.renderers import HTMLRenderer
from apistar import annotate

@annotate(renderers=[HTMLRenderer()])
def hello(username: str, templates: Templates):
    index = templates.get_template('index.html')
    return index.render(username=username)
```
works this way, is this by design?

**Comment by tomchristie** [comment]
See the 0.3.0 release notes, yes. Although there might be some rethinking, not sure yet. See also https://discuss.apistar.org/t/version-0-3-0-released/231/2

**Comment by thomasmktong** [comment]
Seems static files have a similar issue, any idea how to workaround this? Thanks.

![image](https://user-images.githubusercontent.com/7279232/30519617-87e4c198-9bcd-11e7-9513-3ee6fedf3d05.png)

**Comment by tomchristie** [comment]
@thomasmktong - How do you replicate that? Installing, running `apistar new .` and `apistar run` presents the static files correctly in the docs for me.

**Comment by thomasmktong** [comment]
hi @tomchristie, i believe it sometimes works but sometimes doesn't. You may refresh the following link a few times then you should be able to get my error.

https://gatecoin-automation.herokuapp.com/static/css/balance.css

Thanks!

**Comment by tomchristie** [comment]
> Seems static files have a similar issue

Resolved in 0.3.4.

We need to improve (simplify) the logic around when we invoke rendering for responses, but the fix will do for now. (Issue was around the behavior for 304 Not Modified responses)
